src/ResourceManagement/Network/Network.sln

```diff
@@ -3,8 +3,12 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Azure.Management.Compute", "..\Compute\Microsoft.Azure.Management.Compute\Microsoft.Azure.Management.Compute.xproj", "{CBE97730-45F5-448E-88E9-55DF94D65B77}"
+EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Azure.Management.Network", "Microsoft.Azure.Management.Network\Microsoft.Azure.Management.Network.xproj", "{4F22B186-DEB4-4B95-BA58-0187F0E32993}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Azure.Management.Storage", "..\Storage\Microsoft.Azure.Management.Storage\Microsoft.Azure.Management.Storage.xproj", "{A9B44E1B-C26F-44AB-B13F-F22742371125}"
+EndProject
```

Comment:
@irrogozh why were Compute and Storage added to this file? It looks like you are trying to use them for tests, but you already have a reference to their published packages in the `Network.Tests/project.json`, so you don't need to reference the path to their code in this file.